<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PesqTomografias</class>
 <widget class="QDialog" name="PesqTomografias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Paciente</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edtPaciente">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLimparPaciente">
       <property name="text">
        <string>Limpar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPesqPaciente">
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Aluno</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="edtAluno">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLimparAluno">
       <property name="text">
        <string>Limpar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPesqAluno">
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnConfirmar">
     <property name="text">
      <string>&amp;Confirmar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCancelar">
     <property name="text">
      <string>Cancela&amp;r</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
